azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -314,11 +309,6 @@ private void loadCustomJMXAttributeProcessor(Props props) {
     }
   }
 
-
-  public ProjectLoader getProjectLoader() {
-    return projectLoader;
-  }
-
   public ExecutorLoader getExecutorLoader() {
```

Comment:
Yes. we should. There are a lot of classes initialized in `AzkabanExecServer` and other top level services just to pass arguments around. These wirings can simply be removed by using Guice.